mlflow/R/mlflow/R/python.R

```diff
@@ -23,13 +27,26 @@ python_bin <- function() {
 # Returns path to MLflow CLI, assumed to be in the same bin/ directory as the
 # Python executable
 python_mlflow_bin <- function() {
+  in_env <- Sys.getenv("MLFLOW_BIN")
+  if (file.exists(in_env)) {
+    return(in_env)
+  }
   python_bin_dir <- dirname(python_bin())
-  file.path(python_bin_dir, "mlflow")
+  path <- file.path(python_bin_dir, "mlflow")
+  if (!file.exists(path) && file.exists(Sys.getenv("MLFLOW_PYTHON_BIN"))) {
+    stop("Mlflow executable does not seem to be in the same directory than Python.\n",
```

Comment:
nit: same directory as Python. We could also just say "Could not find MLflow CLI executable, please set ..."